tests/acceptance/test_api.py

```diff
@@ -1,27 +1,4 @@
 from sentry.testutils.cases import AcceptanceTestCase
-from sentry.testutils.silo import no_silo_test
-
-
-@no_silo_test(stable=True)
-class ApiTokensTest(AcceptanceTestCase):
-    def setUp(self):
-        super().setUp()
-        self.org = self.create_organization(name="Rowdy Tiger Rowdy Tiger Rowdy Tiger", owner=None)
-        self.project = self.create_project(
-            organization=self.org, teams=[self.team], name="Bengal Bengal Bengal Bengal"
-        )
-        self.login_as(self.user)
-        self.path = "/api/"
-
-    def test_simple(self):
```

Comment:
Deleted this test b/c it's not possible to click the `Create Button` anymore so it would fail. Also, this is only testing that an API is called, so it's not very useful.

Reply:
i'm unsure if leaving this change in and removing the `Scope: Backend` label will cause problems because this is a backend change in a frontend PR.

i don't think the change will get deployed because of the lack of a label

Reply:
Moved to https://github.com/getsentry/sentry/pull/56537